### ISSUE TITLE
Custom Themes Buggy

### ISSUE THREAD

**Issue by dishad** [Marak/colors.js#226]
It seems that the current version of `colors` has made the use of custom themes buggy. I believe that this started occurring for versions > `1.2.1`.

Example:
```
var colors = require('colors');
colors.setTheme({
  warn: 'yellow',
  error: 'red',
  success: 'green',
  info: 'blue',
  exit: 'bold'
});


console.log('Hello success!'.success);
console.log('Hello error!'.error);
console.log('Hello warn!'.warn);
console.log('Hello info!'.info);
console.log('Hello exit!'.exit);

console.log('Hello green!'.green);
console.log('Hello red!'.red);
console.log('Hello yellow!'.yellow);
console.log('Hello blue!'.blue);
console.log('Hello bold!'.bold);
```

will result in:
<img width="299" alt="screen shot 2018-05-04 at 3 00 17 pm" src="https://user-images.githubusercontent.com/13590690/39647204-0a8d4be6-4fac-11e8-8df8-3809f0799c99.png">

**Comment by Marak** [comment]
@DABH ?

**Comment by DABH** [comment]
This has been fixed in `v1.2.4` via https://github.com/Marak/colors.js/commit/855e342c9e26ab3d7232506cc42c240b7d800bf1 .  The args there are String objects not string literals.  Would be great to get to a place where no String objects appear in the codebase but we should be good for now at least.  Thanks for bringing up this issue!
